### DISCUSSION TITLE
k8-sig's BOM

### DISCUSSION THREAD

**Discussion by dgutson** [oss-review-toolkit/ort#7170] [Q&A] [answered]
Hi, I wanted to ask about the functionality relationship between ORT and https://github.com/kubernetes-sigs/bom:
- can ORT benefit from it with some integration?
- or, are they mutually exclusive by running at the same functional level?

Thanks.

**Comment by sschuberth** [comment, marked as answer]
Looks like the `bom` tool basically accepts two types of input: containers and directories / files. Only if the directory happens to contain a Go module, package-level dependencies are determined. I.e. `bom` completely lacks ORT's package-manager support. If the directory is not a Go module, its files are being listed as part of the SPDX BOM.

So bottom line, `bom` rather seems to complement ORT than offering the same functionality, and eventually `bom` could be used to implement https://github.com/oss-review-toolkit/ort/issues/1833.

**Reply by dgutson** [reply]
Thanks, you might want to update https://github.com/oss-review-toolkit/ort/issues/1833#issue-498214314 then.
